Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
     "crates/sui-open-rpc",
     "crates/sui-open-rpc-macros",
     "crates/sui-storage",
+    "crates/sui-testsuite",
```

Comment:
lets not call this a testsuite as we probably want to reserve that for an actual testsuite as this seems to be more about checking a cluster's health than an actual suite of tests to run at ci time.

Reply:
I'm actually thinking about extending this into a comprehensive CI suite. I will read your PR and maybe sync offline on how to move forward. 
For now I will rename it to "sui-cluster-test"